project/api/models/account.py

```diff
@@ -139,7 +139,6 @@ class Account(models.Model):
         "self", related_name="followings", symmetrical=False
     )
 
-    beta_access = models.BooleanField(default=False)
```

Comment:
:warning: **Note** Remember to run the [`makemigrations` command](https://docs.djangoproject.com/en/3.2/ref/django-admin/#django-admin-makemigrations) after changing model fields.